cashu/core/crypto/nut19.py

```diff
@@ -0,0 +1,34 @@
+from hashlib import sha256
+from typing import List
+
+from .secp import PrivateKey, PublicKey
+
+from ..base import BlindedMessage
+
+
+def construct_message(quote_id: str, outputs: List[BlindedMessage]) -> bytes:
+    serialized_outputs = bytes.fromhex("".join([o.B_ for o in outputs]))
```

Comment:
```suggestion
    serialized_outputs = bytes.fromhex("".join([o.B_ for o in outputs]))
```

This should be utf-8 encoded?